src/lib/backend.ts

```diff
@@ -121,9 +104,10 @@ export class ElectronBackend implements Backend {
    */
   public async install(): Promise<boolean> {
     let success = true;
+    const cachePath = join(getApp().getPath('userData'), 'sentry');
```

Comment:
This could go in a utility `getCachePath()` so that we can call it more easily from different places.